drake/CMakeLists.txt

```diff
@@ -326,6 +322,12 @@ if (MATLAB_FOUND)
     PROPERTIES TIMEOUT 300)
 endif()
 
+option(BUILD_DOCUMENTATION "Build Doxygen and Sphinx documentation." ON)
```

Comment:
in fact, after some quick testing, this does some very expensive sphinx build crawls every time that I run make in the drake directory.  that means build times go way way up for everyone during development.  no way we can accept as is.

It would also be nice if the "documentation" target forced the install (to build/share/doc).  The scripts I use to host the documentation on drake002 are easy to update, but they don't actually compile the rest of drake... only the documentation.  Would prefer to keep it that way.